tools/travis_python_watchdog.sh

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+HERE="`dirname \"$0\"`"				# relative
```

Comment:
Most of the scripts of this file is same as travis_mvn_watchdog.sh. The only difference is how to execute the tests. What about moving the changes here to travis_mvn_watchdog.sh and do some refactoring of travis_mvn_watchdog.sh to make it work for both java and python? Maybe we should also rename travis_mvn_watchdog.sh to travis_watchdog.sh?

Reply:
That makes sense. I have combined them in the new commit.